common.yml

```diff
@@ -1,13 +1,11 @@
+
 service:
```

Comment:
We came up with this because I thought ``paymentsservice`` was a little bit annoying. You can't use underscores or dashes. Calling it ``service`` at least links it to the repo somewhat.